.github/actions/build-lambda/action.yml

```diff
@@ -73,14 +73,4 @@ runs:
         if [[ -z $s3Key ]]; then
             s3Key="${{ github.sha }}.zip"
         fi
-        aws s3 cp ~/${{ github.sha }}.zip s3://${{inputs.s3-bucket}}/${s3Key}
-    - uses: actions/upload-artifact@v4
-      if: inputs.s3-key == ''
-      with:
-        name: ${{inputs.scope}}
```

Comment:
Artifact names cant have `/` which they now do. We weren't using this artifact anyway so just removing.